docs/tests.rst

```diff
@@ -110,6 +110,57 @@ Via Docker with a graphical instance of Firefox (macOS)
 
   $ docker-compose -f docker-compose.test.yml run --rm acceptance
 
+How to Debug the Selenium Tests on Docker
+-----------------------------------------
+In order to debug code in the `web` container with the Docker-Selenium test suite it is necessary to use `remote-pdb`_.
```

Comment:
Yeah, but why? Better:

Unlike the other test suites, the container that is running the test code of the
``acceptance`` test suite is different from the one running the application
code. Therefore, in order to debug a test failure, we must connect remotely to
this other container. The tool to achieve this is called `remote-pdb`_.